Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -78,7 +78,7 @@ open class MapView: UIView {
     private let viewAnnotationContainerView = SubviewInteractionOnlyView()
 
     /// Resource options for this map view
-    internal private(set) var resourceOptions: ResourceOptions!
+    internal let resourceOptions: ResourceOptions!
```

Comment:
Seems that implicitly unwrapped optional can be dropped from the type declaration